tox.ini

```diff
@@ -18,21 +18,10 @@ envlist =
 ; this section contains constants that can be referenced elsewhere
 [constants]
 
-; The commit in opentelemetry-python repo that the code in this repository is
-; built against. Use this for pre-GA development, and eventually we will build
-; against releases and remove this. For now, bump this frequently with every PR
-; to develop as closely with that repo as possible while still tracking the
-; actual hash.
-opentelemetry_python_commit = 545068d6eb61beaab031af45b1572d70e6652bc3
-opentelemetry_api_dep =
-  git+https://github.com/open-telemetry/opentelemetry-python.git@{[constants]opentelemetry_python_commit}#egg=opentelemetry-api&subdirectory=opentelemetry-api
-opentelemetry_sdk_dep =
-  git+https://github.com/open-telemetry/opentelemetry-python.git@{[constants]opentelemetry_python_commit}#egg=opentelemetry-sdk&subdirectory=opentelemetry-sdk
-
 base_deps =
   -c dev-constraints.txt
-  {[constants]opentelemetry_api_dep}
-  {[constants]opentelemetry_sdk_dep}
+  opentelemetry-api
```

Comment:
Do these still need to be here if they're already in `dev-constraints.txt`?

Reply:
Yup, the constraint file (`-c` instead of `-r`) just specifies versions, not concrete dependencies. Pip just ignores lines in constraint files that aren't being installed